src/test/kotlin/uk/gov/justice/hmpps/offendersearch/services/MatchServiceTest.kt

```diff
@@ -48,354 +48,7 @@ internal class MatchServiceTest {
 
   @BeforeEach
   fun setUp() {
-    service = MatchService(searchClient, objectMapper, "1")
```

Comment:
Most tests in this class are now covered by integration tests, so they have been removed rather than converting to v2 of the schema.  Kept tests that still add value